ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -94,6 +94,27 @@ if [[ "${RUN_INTEGRATION_TESTS}" == "yes" || \
 
   # shellcheck disable=SC1091
   source "${INTEGRATION_TESTS_CONFIG}"
+
+  # Changing the PATH disables the Bazel cache, so use an absolute path.
+  readonly GCLOUD="/usr/local/google-cloud-sdk/bin/gcloud"
+
+  "${GCLOUD}" --quiet auth activate-service-account --key-file \
+      "${GOOGLE_APPLICATION_CREDENTIALS}"
+  # This is used in a Bigtable example showing how to use access tokens to
+  # create a grpc::Credentials object. Even though the account is deactivated
+  # for use by `gcloud` the token remains valid for about 1 hour.
+  ACCESS_TOKEN="$("${GCLOUD}" --quiet auth print-access-token)"
+  readonly ACCESS_TOKEN
+
+  # Extract the service account name so we can deactivate it.
+  GOOGLE_APPLICATION_CREDENTIALS_ACCOUNT="$(sed -n \
+      's/.*"client_email": "\(.*\)",.*/\1/p' \
+      "${GOOGLE_APPLICATION_CREDENTIALS}")"
+  readonly GOOGLE_APPLICATION_CREDENTIALS_ACCOUNT
+
+  # Deactivate the recently activated service account to prevent accidents.
+  "${GCLOUD}" --quiet auth revoke "${GOOGLE_APPLICATION_CREDENTIALS_ACCOUNT}"
```

Comment:
Q.  Will the change from `revoke --all` save me from losing my `gcloud auth login` credentials, which has been happening when I run tests recently.

Reply:
It will keep your credentials, but still lose the default account.  In other words, you may need to do `gcloud auth set account <your-account-here>` which is less annoying than a full login, but still annoying.

Reply:
I don't see a `gcloud auth set account`.

Might `--configuration config-name` (or `CLOUDSDK_ACTIVE_CONFIG_NAME=config-name`) be useful?

Reply:
Doh! `gcloud` **config** `set account <your-account-here>`.  I will read about that config thing.